icons/sparkle.json

```
@@ -0,0 +1,16 @@
+{
+    "$schema": "../icon.schema.json",
+    "tags": [
+      "sparkle",
+      "star",
+      "shiny",
+      "glitter",
+      "twinkle",
+      "night",
+      "celebration"
+    ],
+    "categories": [
+      "devices"
```

Probably a good idea to change this.

---

so what categories should I add?

---

I'm not the best person to ask this :)

```suggestion
      "shapes"
```